migrations/2024-09-24-205353_uploads/down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE uploads
```

Comment:
nit: could also do same here:
```suggestion
DROP TABLE IF EXISTS uploads
```